Tests/TuistKitTests/Services/Cloud/CloudInitServiceTests.swift

```diff
@@ -56,4 +72,57 @@ final class CloudInitServiceTests: TuistUnitTestCase {
         cloud: .cloud(projectId: "slug", url: "https://cloud.tuist.io/")
         """)
     }
+
+    func test_cloud_init_when_config_does_not_exist() async throws {
+        // Given
+        var content: String?
+        configLoader.locateConfigStub = { _ in
+            nil
+        }
+        fileHandler.stubWrite = { stubContent, _, _ in
+            content = stubContent
+        }
+        createProjectService.createProjectStub = { _, _, _ in
+            "slug"
+        }
```

Comment:
```suggestion
        configLoader.locateConfigStub = { _ in nil }
        fileHandler.stubWrite = { stubContent, _, _ in content = stubContent }
        createProjectService.createProjectStub = { _, _, _ in "slug" }
```

Reply:
I can keep the change but I think the multiline versions are more readable